alerts/src/commonMain/kotlin/Alerts.kt

```diff
@@ -210,28 +260,28 @@ data class Alert(
 }
 
 /**
- * Interface that defines actions that can be applied to the alert.
+ * Interface that defines actions that can be used for presenting an [Alert].
  */
 interface AlertActions {
     /**
-     * Presents an alert
+     * Presents an [Alert]
      *
      * @param animated Pass `true` to animate the presentation
      * @param completion The block to execute after the presentation finishes
      */
     fun showAsync(animated: Boolean = true, completion: () -> Unit = {})
 
     /**
-     * Presents an alert and suspends
+     * Presents an [Alert] and suspends until completion
      *
-     * @param animated
-     * @return The action that was performed by button click
-     *         or `null` if the alert was cancelled by pressing back (on Android)
+     * @param animated Pass `true` to animate the presentation
+     * @return The [Alert.Action] that was performed by button click
+     *         or `null` if the alert was cancelled by the user.
```

Comment:
I worth to mention, that cancelling means pressing back button on Android, as on iOS dismiss alert possible only by pressing any action button